profiles/use.desc

```diff
@@ -172,8 +175,9 @@ livecd - !!internal use only!! DO NOT SET THIS FLAG YOURSELF!, used during livec
 llvm-libunwind - Use sys-libs/llvm-libunwind instead of sys-libs/libunwind
 lm-sensors - Add linux lm-sensors (hardware sensors) support
 lua - Enable Lua scripting support
-lzma - Support for LZMA (de)compression algorithm
 lz4 - Enable support for lz4 compression (as implemented in app-arch/lz4)
+lzip - Enable support for lzip compression
```

Comment:
Why not (de) here too?

Reply:
Sounds like an overkill to me.

Reply:
Then why are we doing it at all for other formats? Clean it up in a new commit?